<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wJob</class>
 <widget class="QWidget" name="wJob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Con el Trabajo Actual</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="otherJobButton">
          <property name="text">
           <string>Clonar Trabajo (Otro Trabajo, misma personas)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sameWorkButton">
          <property name="text">
           <string>Clonar Tab (Mismo, diferente Personas)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="differentButton">
          <property name="text">
           <string>Agregar Tabs (Diferente, diferentes Personas)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Para el:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="fechaLabel">
            <property name="text">
             <string>Fecha:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="fechaDateEdit">
            <property name="displayFormat">
             <string>dddd dd/MM</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="horaLabel">
            <property name="text">
             <string>Hora:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTimeEdit" name="horaTimeEdit">
            <property name="currentSection">
             <enum>QDateTimeEdit::HourSection</enum>
            </property>
            <property name="displayFormat">
             <string>H:mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Trabajos</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <item row="2" column="0">
           <widget class="QRadioButton" name="ownButton">
            <property name="text">
             <string>Propio:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="bookButton">
            <property name="text">
             <string>Libro:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="fileButton">
            <property name="text">
             <string>Archivo:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="bookLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="fileLineEdit"/>
          </item>
          <item row="2" column="1">
           <layout class="QVBoxLayout" name="ownVerticalLayout">
            <item>
             <widget class="wBooksWidget" name="ownWidget" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Operaciones:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="markFinishedButton">
          <property name="text">
           <string>Marcar Como Terminado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="discardButton">
          <property name="text">
           <string>Descartar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reprintButton">
          <property name="text">
           <string>Reimprimir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreCheckBox">
          <property name="text">
           <string>Ignorar Completo</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wBooksWidget</class>
   <extends>QWidget</extends>
   <header>wbookswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bookButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>bookLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>fileLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ownButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ownWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
